vscode/.gitignore

```diff
@@ -0,0 +1,4 @@
+out
```

Comment:
Shall we add all git ignore codes to root .gitignore file?

Reply:
Done